app/src/modules/autoUpdater.js

```diff
@@ -12,17 +13,17 @@ const getErrorMessage = (error) => {
 export default ({ // eslint-disable-line max-statements
   autoUpdater, dialog, win, electron,
 }) => {
+  autoUpdater.logger = logger;
```

Comment:
maybe remove the auto updater changes from this PR?

Reply:
Removed. It was a problem with the branch base.